### PR TITLE
Improved Docker Compose

### PR DESCRIPTION
**Pull request by alexander-torosh**
Improved Docker usage. Created main docker-compose.yml file. Updated documentation for developers. Used Node.js v8 instead of v6 to support sync/async feature.

### CODE REVIEW

**Comment by alexander-torosh** [comment]
Hello @wobbals I made different improvements for better and easier usage with Docker Compose and used Node.js v8 instead of v6. I tested the app and it's working well.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - "redis"
+    environment:
+      - REDIS_HOST=redis
+
+  barc-ws:
+    build: "./webapp/"
+    volumes:
+      - "./webapp/bin/www:/src/bin/www"
+    ports:
+      - "11500:443"
```
Comment (wobbals) [inline docker-compose.yml:20]: Where is this internal port number coming from? I don't see any change to the default used in this PR.

Comment (alexander-torosh) [reply docker-compose.yml:20]: @wobbals I used port 11500 only for development usage. 
This is an absolutely random port number which shouldn't conflict with other ports on developer's development machine.

Added instructions for developers:

* Open `http://localhost:11500/barc` in your browser to check working
* Use `http://localhost:11500/barc/job` to put your jobs

Comment (alexander-torosh) [reply docker-compose.yml:20]: Docker links guest port 11500 (developer's machine) with host port 443 (inside Docker container)

Comment (wobbals) [reply docker-compose.yml:20]: Ah, my mistake. I was reading `CONTAINER:HOST` rather than `HOST:CONTAINER` for the port declaration syntax.

**Comment by wobbals** [comment]
* Does this PR deprecate the older docker compose file at https://github.com/wobbals/barc/blob/master/webapp/docker-compose.yml

* Can we keep the original container names instead of using a demio prefix for the master branch?

Thanks!

**Comment by alexander-torosh** [comment]
@wobbals 

- Yes, we really don't need that anymore, because I merged everything to single docker-compose.yml file which contains both Docker containers.

- Sorry, my bad 😄  updated to original names

**Comment by wobbals** [comment]
Thanks for doing this!

**Pull request merged**